chromium_src/chrome/browser/k_anonymity_service/k_anonymity_trust_token_getter.h

```diff
@@ -0,0 +1,16 @@
+/* Copyright (c) 2025 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_CHROMIUM_SRC_CHROME_BROWSER_K_ANONYMITY_SERVICE_K_ANONYMITY_TRUST_TOKEN_GETTER_H_
+#define BRAVE_CHROMIUM_SRC_CHROME_BROWSER_K_ANONYMITY_SERVICE_K_ANONYMITY_TRUST_TOKEN_GETTER_H_
+
+#define TryGetTrustTokenAndKey(...)    \
```

Comment:
aren't we already overriding or otherwise disabling InterestGroupManager? In either case it seems like we should override `KAnonymityServiceFactory::GetForProfile` to return nullptr?

Reply:
Removed in https://github.com/brave/brave-core/pull/26537/commits/8da062a9dd5808f17b6e8c8553578108c8f29b85